README.md

```diff
@@ -121,26 +126,142 @@ packages](./docs/how_to_write_packages.md).
 
 ![Preflight package structure diagram](./docs/images/preflight_package.png)
 
-## Get Preflight
+## Installation
+
+This section outlines how to install Preflight.
+
+### Agent
+
+The following instructions walk through the installation of the Preflight agent
+to gather data about cluster pods and send them to the backend for analysis.
+
+**To complete the secret manifest below, you will need to have a Preflight
+token.**
+
+First create a namespace for the preflight components:
```

Comment:
I feel like these manifests should be in files in `examples/` or `deploy/` so they are easier for user to consume, though this might be an issue for a future PR.

Reply:
I think my other comment touches on this.